src/mo_frontend/printers.ml

```diff
@@ -68,184 +91,188 @@ let string_of_symbol = function
+  | X (N N_exp_ob_) -> "<exp(ob)>", eg_exp
+  | X (N N_exp_bin_bl_) -> "<exp_bin(bl)>", eg_exp
+  | X (N N_exp_bin_ob_) -> "<exp_bin(ob)>", eg_exp
+  | X (N N_exp_obj) -> "<exp_obj>", eg_exp_obj
+  | X (N N_exp_nest) -> "<exp_nest>", eg_exp
+  | X (N N_block) -> "<block>", "{}"
+  | X (N N_exp_field) -> "<exp_field>", eg_exp_field
+  | X (N N_exp_nondec_bl_) -> "<exp_nondec(bl)>", eg_exp
+  | X (N N_exp_nondec_ob_) -> "<exp_nondec(ob)>", eg_exp
+  | X (N N_exp_nonvar_bl_) -> "<exp_nonvar(bl)>", eg_exp
+  | X (N N_exp_nonvar_ob_) -> "<exp_nonvar(ob)>", eg_exp
+  | X (N N_exp_nullary_bl_) -> "<exp_nullary(bl)>", eg_exp
+  | X (N N_exp_nullary_ob_) -> "<exp_nullary(ob)>", eg_exp
+  | X (N N_exp_plain) -> "<exp_plain>", "true"
+  | X (N N_exp_post_bl_) -> "<exp_post(bl)>", eg_exp
+  | X (N N_exp_post_ob_) -> "<exp_post(ob)>", eg_exp
+  | X (N N_obj_or_class_dec) -> "<obj_or_class_dec>", "object " ^ eg_pat ^ " = {}"
+  | X (N N_option_exp_post_ob__) -> "<exp_post(ob)>?", eg_exp
+  | X (N N_exp_un_bl_) -> "<exp_un(bl)>", eg_exp
+  | X (N N_exp_un_ob_) -> "<exp_un(ob)>", eg_exp
+  | X (N N_func_body) -> "<func_body>", "{}"
+  | X (N N_imp) -> "<imp>", eg_imp
+  | X (N N_import_list) -> "<import_list>", eg_imp
+  | X (N N_inst) -> "<inst>", "<" ^ eg_typ ^ ">"
+  | X (N N_lit) -> "<lit>", "true"
+  | X (N N_ob) -> "<ob>", eg_exp_obj
+  | X (N N_obj_body) -> "<obj_body>", "{}"
+  | X (N N_option_EQ_) -> "=?", "=?"
+  | X (N N_option_exp_nullary_ob__) -> "<exp_nullary(ob)>?", eg_exp
+  | X (N N_option_typ_args_) -> "<typ_args>?", eg_typ_args
+  | X (N N_option_query_) -> "<query>?", "query"
+  | X (N N_parse_module_header) -> entry_point "<parse_module_header>"
+  | X (N N_parse_prog) -> entry_point "<parse_prog>"
+  | X (N N_parse_prog_interactive) -> entry_point "<parse_prog_interactive>"
+  | X (N N_parse_stab_sig) -> entry_point "<parse_stab_sig>"
+  | X (N N_pat) -> "<pat>", eg_pat
+  | X (N N_pat_bin) -> "<pat_bin>", eg_pat
+  | X (N N_pat_field) -> "<pat_field>", eg_pat ^ " " ^ eg_pat
+  | X (N N_pat_nullary) -> "<pat_nullary>", eg_pat
+  | X (N N_pat_plain) -> "<pat_plain>", eg_pat
+  | X (N N_pat_un) -> "<pat_un>", eg_pat
+  | X (N N_path) -> "<path>", "A.B.C"
+  | X (N N_annot_opt) -> "<annot_opt>", eg_annot
+  | X (N N_seplist_case_semicolon_) -> seplist ("<case>", eg_case) semi
+  | X (N N_seplist_dec_SEMICOLON_) -> seplist ("<dec>", eg_dec) semi2
+  | X (N N_seplist_dec_semicolon_) -> seplist ("<dec>", eg_dec) semi
+  | X (N N_seplist_typ_dec_semicolon_) -> seplist ("<typ_dec>", eg_typ_dec) semi
+  | X (N N_seplist_dec_field_semicolon_) -> seplist ("<dec_field>", eg_dec_field) semi
+  | X (N N_seplist_exp_ob__COMMA_) -> seplist ("<exp(ob)>", eg_exp) comma
+  | X (N N_seplist_exp_field_semicolon_) -> seplist ("<exp_field>", eg_exp_field) semi
+  | X (N N_seplist1_exp_field_semicolon_) -> "seplist1(<exp_field>,<semicolon>)", eg_exp_field
+  | X (N N_separated_nonempty_list_AND_exp_post_ob__) -> "seplist+(<exp_post(ob)>,and)", eg_exp ^ "and"
```

Comment:
```suggestion
  | X (N N_separated_nonempty_list_AND_exp_post_ob__) -> "seplist+(<exp_post(ob)>,and)", eg_exp ^ " and"
```

Reply:
(I got weird output with "{;" in the interpreter with a trailing "and"

Reply:
yeah, it was wrong, fixed